.github/workflows/crowdin.yml

```diff
@@ -0,0 +1,79 @@
+name: Crowdin Action
+
+on:
+  push:
+    branches: [ develop ]
+
+jobs:
+  synchronize-with-crowdin:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: crowdin action
+      uses: crowdin/github-action@1.0.16
+      with:
+        upload_translations: true # default is false
+        # Use this option to upload translations for a single specified language
+        download_translations: true
+        # Upload sources to Crowdin
+        # upload_sources: # default is true
+        # Upload translations to Crowdin
+        # upload_language: # optional
+        # Automatically approves uploaded translations
+        # auto_approve_imported: # default is false
+        # Defines whether to add translation if it is equal to source string in Crowdin project
+        # import_eq_suggestions: # default is false
+        # Make pull request of Crowdin translations
+        # Use this option to download translations for a single specified language
+        # download_language: # optional
+        # Skip untranslated strings in exported files (does not work with .docx, .html, .md and other document files)
+        skip_untranslated_strings: true
```

Comment:
```suggestion
        skip_untranslated_strings: false
```

Reply:
@VaiTon would like your input on that one

Reply:
Let it be false, let's see its behavior